R/super.R

```diff
@@ -36,20 +36,80 @@
 super <- function(...) {
   # TODO: (RK) Error on anonymous function calls.
   fn <- as.character(sys.call(-1)[[1]])
-
-  env <- parent.frame(2)
+  if (identical(fn[1], "get")) {
+    if (length(fn) == 1) { # TODO: (RK) Actually parse out super::multi_parent_env
+      stop("super::super does not support ", sQuote("get"), " calls ",
+           "due to internal purposes.")
+    } else {
+      # From the magic get() call below, we extract the function name.
+      fn <- fn[2] 
+      env <- parent.frame()
```

Comment:
`parent.frame()` because the `eval.parent(substitute(...))` below ensures we're no longer in a `super::super` call directly.